documentation/modules/upgrading/proc-downgrade-cluster-operator.adoc

```diff
@@ -17,8 +17,12 @@ This procedure describes how to downgrade a Cluster Operator deployment to a pre
 
 .Before you begin
 
-Check the downgrade requirements of the xref:ref-operator-cluster-feature-gates-{context}[Strimzi feature gates].
-If a feature gate is permanently enabled, you may need to downgrade to a version that allows you to disable it before downgrading to your target version. 
+Check the xref:ref-operator-cluster-feature-gates-{context}[Strimzi feature gate releases].
+If a feature gate is enabled, you may need to disable it before downgrading. 
+If the feature gate is permanently enabled, you may need to first downgrade to a version where it can still be disabled.
+
+Kafka clusters managed by Strimzi always use node pools. 
+Downgrading to a cluster without node pools is not possible.
```

Comment:
I'm not sure I would understand as a user what to look for and what to do about it to be honest. I would expect this to have a more clear information about the particular feature gates and their upgrade/downgrade limitations.

I would either remove it completely right now or leave some generic statement that _sometimes, feature gates might need to be disabled when upgrading / downgrading. But that right now there is nothing special like that._.

Reply:
I think it might be useful to reassure the user that they don't need to worry about feature gates at the moment, so I've adjusted the statement.